internal/ui/app.go

```diff
@@ -146,6 +148,12 @@ func (a *App) bindKeys() {
 		tcell.KeyCtrlU: NewSharedKeyAction("Clear Filter", a.clearCmd, false),
 		tcell.KeyCtrlQ: NewSharedKeyAction("Clear Filter", a.clearCmd, false),
 	}
+
+	if a.Config.K9s.AllowSuspend && runtime.GOOS != "windows" {
```

Comment:
I think it might be best to make a call on the config and keep this functionality encapsulated vs exposing on the call site? Perhaps something  a.Config.K9s.IsSuspendable()?

Reply:
yep makes sense - done :)